### ISSUE TITLE
kblad rhabdosphincter is mapped to both male and female ontology ids

### ISSUE THREAD

**Issue by tgbugs** [open-physiology/apinatomy-models#11]
@MCSZ @stappan did we upstream rhabdosphincter, it doesn't seem to have been swapped out for a more generic version.

**Comment by MCSZ** [comment]
@tgbugs No, this terms is not in the [upstream term request sheet](https://docs.google.com/spreadsheets/d/1-cOSWVusX8HYu06jidqkUhioMby4wpJv7ePnLpbcOho/edit#gid=1021950839) but was added to [InterLex at the beginning of the year ](http://uri.interlex.org/base/ilx_0793141). @stappan How can I add this term to your pipeline?

**Comment by stappan** [comment]
I don't recall seeing that term on my list for UBERON , I can check when I return to office

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
From: Tom Gillespie ***@***.***>
Sent: Tuesday, June 21, 2022 7:11:54 PM
To: open-physiology/apinatomy-models ***@***.***>
Cc: Sue Tappan ***@***.***>; Mention ***@***.***>
Subject: [open-physiology/apinatomy-models] kblad rhabdosphincter is mapped to both male and female ontology ids (Issue #11)


@MCSZ<https://github.com/MCSZ> @stappan<https://github.com/stappan> did we upstream rhabdosphincter, it doesn't seem to have been swapped out for a more generic version.
